modules/ray_tracing/doc/content/modules/ray_tracing/examples/line_integrals.md

```diff
@@ -14,17 +14,17 @@ We begin with the standard "simple diffusion" problem:
 For this problem, we seek the value of the integral
 
 !equation
-\int_L u_h~dr, \quad L = \{\vec{r}_1 + t\vec{r}_2 \mid t \in [0, 1]\}
+\int_L u_h(\vec{r})~dr, \quad L = \{\vec{r}_1 + t\vec{r}_2 \mid t \in [0, 1]\}
 
 for the lines defined by
 
 !equation
-\vec{r}_1 = (0, 0)\,, \quad \vec{r}_1 = (5, 5)
+\vec{r}_1 = (0, 0)\,, \quad \vec{r}_2 = (5, 5)
 
 and
 
 !equation
-\vec{r}_1 = (5, 0)\,, \quad \vec{r}_1 = (5, 5)
+\vec{r}_1 = (5, 0)\,, \quad \vec{r}_2 = (5, 5)
 
 For simplicity, we will denote the lines as `diag` and `right_up`, respectively.
```

Comment:
It's possible that the Diffusion Kernel is confusing in this example (a reader might think "but isn't the Kernel going to change u?"), but i can't see a nice way of avoiding that.  Perhaps you could also include the statement that u(x, y, t) = x/5.

Reply:
Let me know what you think about 7f0dfebcbf4e81857da35401e395da156b830cee:

![Screen Shot 2020-12-16 at 3 47 00 PM](https://user-images.githubusercontent.com/22062133/102415749-f2332280-3fb5-11eb-8f3e-4bee207c8bd1.png)